wger/gym/templates/gym/member_list.html

```diff
@@ -26,7 +26,9 @@ <h4>{% trans "Administrators and trainers" %}</h4>
     <th style="width: 10%;">{% trans "ID" %}</th>
     <th style="width: 40%;">{% trans "Username" %}</th>
     <th>{% trans "Name" %}</th>
+    <th>{% trans "Active" %}</th> {% comment %} Header showing if user is active or not {% endcomment %}
```

Comment:
Good implementation on writing comments

Reply:
Nice work on the comment, it makes it easy to follow through the code.